src/modules/subscribers.ts

```diff
@@ -9,6 +9,15 @@ function updateNulls(entity: any) {
   }
 }
 
+function updateUndefined(entity: any) {
+  if (entity) {
+    const that: any = entity;
+    Object.keys(entity).forEach(k => {
+      if (that[k] === undefined) that[k] = null;
+    });
+  }
+}
+
```

Comment:
```suggestion
function updateUndefined(entity: any) {
  if (entity) {
    Object.keys(entity).forEach(k => {
      if (entity[k] === undefined) entity[k] = null;
    });
  }
}
```
I think this does the same thing. Is there a reason why you have created the `that` variable?

Reply:
I think you are right, this is copied from the other method in this file and came from a legacy code inside the entities' class code, which was messing up with `this`. I'll try to update it in a follow-up PR to unblock PRs waiting for this one.